test/test_utils.go

```diff
@@ -281,3 +287,30 @@ func verifySignature(ctx context.Context, t *testing.T, c *clients, obj objects.
 		}
 	}
 }
+
+// restartChainsControllerPod restarts the pod running Chains
+// it then waits for a given timeout for the pod to resume running state
+func restartChainsControllerPod(c kubernetes.Interface, timeout time.Duration) error {
```

Comment:
Should this take a context? It looks like the caller, setConfigMap, already has one to pass it in.

Reply:
Fixed